src/index.ts

```diff
@@ -1,20 +1,45 @@
-import express, { Request, Response, NextFunction} from "express";
+import express, { Request, Response, NextFunction } from "express";
 import path from "path";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
 
 const app = express();
 
 app.use(express.static(path.join(__dirname, "../public")));
+app.use(express.json());
 
 app.get("/", (req: Request, res: Response, next: NextFunction): void => {
-    try {
-        res.send("index.html");
-    } catch (error) {
-        next(error);
-    }
-})
+  try {
+    res.send("index.html");
+  } catch (error) {
+    next(error);
+  }
+});
+
+app.get("/users", async (req, res, next) => {
+  try {
+    const users = await prisma.user.findMany();
+    res.send({ users });
+  } catch (e) {
+    next(e);
+  }
+});
+
+app.post("/users", async (req, res) => {
+  const { email, username, password } = req.body;
+  const user = await prisma.user.create({
+    data: {
+      email,
+      username,
+      password,
+    },
+  });
+  res.send({ user });
+});
```

Comment:
These will be moved into `/api/users` router later right? Find for testing, just maybe add a TODO for moving / deleting later so as to not leave extra routes around that you're not using in the app